### PR TITLE
chore: rename some src folders

### PR DESCRIPTION
**Pull request by quake**
1. [rename src/ckb to src/fiber](https://github.com/nervosnetwork/cfn-node/pull/137/commits/117d66adad754a661e80aae445d09a02cb0da825)
2. [rename src/ckb_chain to src/ckb](https://github.com/nervosnetwork/cfn-node/pull/137/commits/498248693e599f559cc3a53e19b364af00f4d087)
3. [rename CkbConfig to FiberConfig](https://github.com/nervosnetwork/cfn-node/pull/137/commits/e96d4f173b11bf0085b43641e8609d9454d4a8dd)
4. [rename CkbChainConfig to CkbConfig](https://github.com/nervosnetwork/cfn-node/pull/137/commits/fccd90200207c0d929e8f90fa4c1f3c94d247428)

### CODE REVIEW

**Comment by chenyukang** [comment]
rename `src/ckb_chain` to `src/ckb`.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# CKB Fiber Network Node
+# Fiber Network Node
 
-CKB Fiber Network Node (cfn-node) is a reference implementation of a CKB Fiber Network protocol (cfn-protocol). In the current stage, it's not a production-ready implementation, but a prototype to demonstrate the feasibility of the cfn-protocol. It's capable of:
+Fiber Network Node (FNN) is a reference implementation of Fiber Network/Protocol (FNP). In the current stage, it's not a production-ready implementation, but a prototype to demonstrate the feasibility of FNP. It's capable of:
```
Comment (doitian) [inline README.md]: Why slash in "Fiber Network/Protocol"?

File: src/ckb/actor.rs [hunk 2]
```diff
@@ -375,7 +375,7 @@ mod test_utils {
                         request, &tx, &fulfilled_tx
                     );
 
-                    if let Err(e) = reply_port.send(Ok((fulfilled_tx))) {
+                    if let Err(e) = reply_port.send(Ok(fulfilled_tx)) {
```
Comment (chenyukang) [inline src/ckb/actor.rs:378]: why this was not reported by compiler before this commit,
`#[warn(unused_parens)]` should be enabled by default.

Comment (quake) [reply src/ckb/actor.rs:378]: I think it's in test package, I found this error when run `cargo test`

**Pull request merged**